docs/platform/faq.mdx

```diff
@@ -27,15 +27,14 @@ description: Answers to the most common questions about the Platform API and ato
 - An atom is a customizable UI component that handles scheduling on behalf of your user.
 - Everything from the front-end to API calls is being handled by the atom, all you need to is import the atom and drop it in your code.
 
-### 6. What are the minimun setup requirments for using atoms?
-- You need to have a project that uses React version 18 and above.
-- At the moment atoms are only supported in React, so there's no way to use them in vanilla js or any other popular framework.
```

Comment:
I removed the 2nd bullet point because it is not entirely true. Frameworks like Astro support React, and thus should implicitly support atoms as well. Happy to revert this change if you have proof otherwise.